napari_clusters_plotter/_plotter.py

```diff
@@ -349,9 +354,6 @@ def run(
                 self.graphics_widget.pts,
             )
 
-            # get colormap as rgba array
-            from vispy.color import Color
```

Comment:
Local imports and smarter than global imports because the Python code get's loaded faster when starting the plugin...

Reply:
Ah right, let me put it back where it was then :)